### ISSUE TITLE
Show tasks only assigned to that person

### ISSUE THREAD

**Issue by eyalabadi98** [mrsarm/django-coleman#2]
Hello,
I am wondering if anybody can give me some guidance into how to implement a way for non-admins to only see the tasks assigned to me.

Any help appreciated

**Comment by mrsarm** [comment]
Yes @eyalabadi98 , I want to add a filter to the side bar of the Django list view time ago, with a label like _"Assigned to me"_, but I didn't have time to write the code.

I added this post from Stackoverflow to my bookmarks, becuase I think this is the best way to solve it: https://stackoverflow.com/questions/12102697/creating-custom-filters-for-list-filter-in-django-admin/45136544#45136544

I you can do it, any PR would be appreciated.

**Comment by eyalabadi98** [comment]
Hey,
I will definitely want to do it, but I don't want it to be optional. They can only see it it was assigned to me. Kind of my way of doing this is to create a permision first, then add a filter on the backend that only shows the column User.username if it matches the currently logged in user, and if they have the permission for global view, they can see everything. 

I'll hopefully post an update soon

**Comment by mrsarm** [comment]
OK, sounds good, maybe configurable through Django settings to be optional, or even set a name of a group where users are able to see any task not just admin user would be great!

**Comment by eyalabadi98** [comment]
Yes, I got it working, but I created my own filter, so now the default filter sidebar doesnt work and I'm stuck on how to receive and display filters from settings.py. Hopefully I can figure it out
